README.md

```diff
@@ -216,8 +232,38 @@ go get ./...
 go build
 ```
 
+## Event Filtering
+
+The 4.X event forwarder supports custom filter defintions via golang's templating language to filter the stream of Cb Event Messages coming over the event-forwarder.
+
+```
+filter:
+    enabled: true
+    template: >-
+              {{if (eq .type "alert.watchlist.hit.query.binary") -}}
```

Comment:
Replace this with an example that's more "interesting" here - you can already filter by event types... a real-world example would be customers who want to collect more information about certain workstations. So you could have an example where you drop all `ingress.` events *unless* the sensor hostname started with `CEO-` (`CEO-LAPTOP`, `CEO-WORKSTATION`, `CEO-CONFROOM`, etc.)